internal/driver/webui_test.go

```diff
@@ -119,7 +119,15 @@ func TestWebInterface(t *testing.T) {
 		for count := 0; count < 2; count++ {
 			wg.Add(1)
 			go func() {
-				http.Get(path)
+				res, err := http.Get(path)
+				if err != nil {
+					t.Error("could not fetch", c.path, err)
```

Comment:
Add continue, drop else. https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Reply:
Added a returned and called wg.Done() before. 
Done.